cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -1,74 +1,65 @@
-+
-[source,terminal]
-----
-$ odo delete --deploy
-----
-// #5237
-* Adds mapping support to the `odo link` command
-// #5279
-* Supports ephemeral volumes using the `ephemeral` field in `volume` components
-// #5270
-* Sets the default answer to `yes` when asking for telemetry opt-in
-// #5260
-* Improves metrics by sending additional telemetry data to the devfile registry
-// #5287
-* Updates the bootstrap image to `registry.access.redhat.com/ocp-tools-4/odo-init-container-rhel8:1.1.11`
-// #5308
-* The upstream repository is available at link:https://github.com/redhat-developer/odo[]
+== Notable changes and improvements in `{odo-title}` version 3.0.0
 
+`odo` 3.0 includes new commands to improve the developer experience, providing the same functionality as before, but in a faster and more efficient manner.
 
+For more information, see the section "Major highlights of odo 3.0" in xref:./understanding-odo.adoc#understanding-odo[Understanding odo].
 
 [id="odo-fixed-issues_{context}"]
-== Bug fixes
-// #5294
-* Previously, `odo deploy` would fail if the `.odo/env` file did not exist. The command now creates the `.odo/env` file if required.
-// #5286
-* Previously, interactive component creation using the `odo create` command would fail if disconnect from the cluster. This issue is fixed in the latest release.
+== Bug fixes in `{odo-title}` version 3.0.0
 
+* `odo` does not honor directory `.gitignore` entries like `/target`.
 
-[id="odo-getting-support_{context}"]
-== Getting support
+* Port names should match endpoint names.
 
-.For Product
+* `odo deploy` - devfile variable substitution does not occur in URI-referenced deployment manifest YAML file.
 
-If you find an error, encounter a bug, or have suggestions for improving the functionality of `{odo-title}`, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose *OpenShift Developer Tools and Services* as a product type and *odo* as a component.
+* `odo init --name <name> --devfile-path <path>` panics when a non-existent path is used.
 
-Provide as many details in the issue description as possible.
+* Commands should get namespace from `env` file.
 
-.For Documentation
+* `odo dev` fails after component was deployed using `odo deploy`.
 
-If you find an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+* `odo dev` does not use `.gitignore` to sync files to container.
 
+* `odo deploy` and `odo build-images` not working on Windows.
 
+* `odo delete` fails for Kubernetes components.
 
+* Add support for `command` field in container components.
 
+* Stopping `odo dev` also removes `odo deploy` resources.
 
-////
-[id="odo-known-issues_{context}"]
-== Known issues
+* Configuration shows volumes as containers
 
-////
+* Do not save component name in `env.yaml`.
```

Comment:
Is this a known issue? Rewrite to use the format provided in the known issues template.